### ISSUE TITLE
Panic upon closing game

### ISSUE THREAD

**Issue by PerfectlyFineCode** [jbrd/bevy_mod_imgui#20]
**Describe the bug**
When closing the window / game, it will give you a panic message.

**To Reproduce**
1. Start game
2. Close game
3. Upon closing the game you will see a panic message stating that 'no entry found for key'

**Expected behavior**
it should gracefully shut down.

**Screenshots**
![image](https://github.com/jbrd/bevy_mod_imgui/assets/70201838/801dcabb-d68d-4f0c-b6eb-b3e25ea2659a)

**Platform (please complete the following information):**
 - OS: Linux (Ubuntu 22.04 LTS) & Windows 11

**Additional context**
- Version: 0.1.0 (bevy 0.11)
```
thread '<unnamed>' panicked at /home/nautilus/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bevy_mod_imgui-0.1.0/src/lib.rs:545:54:
no entry found for key
stack backtrace:
   0: rust_begin_unwind
             at /rustc/fdaaaf9f923281ab98b865259aa40fbf93d72c7a/library/std/src/panicking.rs:645:5

```

**Comment by jbrd** [comment]
Hi @PerfectlyFineCode - thanks for logging this. I've got a tentative fix for this which I'll get in to a hot fix in the coming days. This is currently authored for version `0.2.0` (Bevy 0.12) - do you need this fix backported to 0.1.0 (Bevy 0.11)?

**Comment by PerfectlyFineCode** [comment]
> Hi @PerfectlyFineCode - thanks for logging this. I've got a tentative fix for this which I'll get in to a hot fix in the coming days. This is currently authored for version `0.2.0` (Bevy 0.12) - do you need this fix backported to 0.1.0 (Bevy 0.11)?

I would love a backport for this since not many crates has updated to 0.12 yet.

**Comment by jbrd** [comment]
Hi @PerfectlyFineCode - this has now been fixed and backported, and published on crates.io.

* 0.2.1 contains the fix for Bevy 0.12
* 0.1.1 contains the fix for Bevy 0.11

Thanks again for reporting this :)
